cfpb/consumerfinance.gov
# Health check

* Adds [`django-health-check==3.15.6`](https://github.com/KristianOellegaard/django-health-check)
* Enabled DB health check
* Accessible @ `/ht/`

---




## Additions

* Added library requirement
* Added URL to cfgov
* Added health check apps to cfgov `INSTALLED_APPS`


## How to test this PR

1. Launch CFGOV with DB
2. Navigate to `/ht/`
3. Verify 200 response with healthy DB check present
3. Stop DB container
4. Refresh `/ht/`
5. Verify 500 response and DB health check shows error
6. Bring DB container back up
7. Verify 200 response with healthy DB check present


## Screenshots
![image](https://user-images.githubusercontent.com/59629491/152893136-b6b6189c-5978-4e38-aac3-5a6a495b5bd6.png)
![image](https://user-images.githubusercontent.com/59629491/152893187-9db1f22c-e4e5-44b4-b558-2019e863366c.png)

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
